tests/functional/i/inconsistent/inconsistent_returns.py

```diff
@@ -387,3 +387,21 @@ def bug_pylint_4122_bis(s):
         return n
     except ValueError:
         parser_error_name('parser error')
+
+
+def bug_pylint_4019(x):
```

Comment:
```suggestion
# https://github.com/PyCQA/pylint/issues/4019
def bug_pylint_4019(x):
```
I like to add a direct link to the issue.

Reply:
Great idea!